### PR TITLE
Cloud Champions articles

### PR DESCRIPTION
**Pull request by gkindavishal**
Fixes b/267254564

### CODE REVIEW

**Comment by 5wathi** [comment]
@gkindavishal : Just the 2 author bio updates. Rest looks good to me.

**Comment by gkindavishal** [comment]
> @gkindavishal : Just the 2 author bio updates. Rest looks good to me.

Hey @5wathi 
I couldn't find any bio for Paul Jones. Can you help me with that?

**Review by 5wathi — CHANGES_REQUESTED** [review]

File: config/authors/rubenszimbres.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Rubens Zimbres",
+  "bio": "",
```
[the remark this answers is not in the document]
Comment (gkindavishal) [reply config/authors/rubenszimbres.json]: Updated this

File: config/authors/paulwilljones.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Paul Jones",
+  "bio": "",
```
[the remark this answers is not in the document]
Comment (gkindavishal) [reply config/authors/paulwilljones.json]: Updated this

**Comment by gkindavishal** [comment]
@5wathi I've updated the bios, please review the changes and merge PR if it looks good to you

**Comment by 5wathi** [comment]
done @gkindavishal

**Pull request merged**